data/lib/src/network/linshare_http_client.dart

```diff
@@ -594,4 +594,15 @@ class LinShareHttpClient {
         data: recipients.map((data) => GenericUserDto(data.mail, lastName: data.lastName, firstName: data.firstName).toJson()).toList().toString());
     return UploadRequestGroupResponse.fromJson(resultJson);
   }
+
+  Future<UploadRequestGroupResponse> archiveUploadRequestGroup(UploadRequestGroupId uploadRequestGroupId) async {
+    final resultJson = await _dioClient.put(
+        Endpoint.uploadRequestGroups
+            .withPathParameter(uploadRequestGroupId.uuid)
+            .withPathParameter(Endpoint.status)
+            .withPathParameter('ARCHIVED')
+            .withQueryParameters([BooleanQueryParameter('copy', true)])
```

Comment:
This is be hard code too. The param `copy` should be passed as a dynamic parameter